docs/uppy-core.mdx

```diff
@@ -1116,6 +1116,8 @@ uppy.on('upload-error', (file, error, response) => {
 #### `upload-retry`
 
 Fired when an upload has been retried (after an error, for example).
+  
+> ⚠️ Note that this method is event is not triggered when the user retries all uploads, it will trigger the `retry-all` event intstead.
```

Comment:
```suggestion
:::note
Note that this method is event is not triggered when the user retries all uploads, it will trigger the `retry-all` event intstead.
:::
```